grpc-example/grpc-example-scala/hello-proxy-impl/src/main/resources/logback.xml

```diff
@@ -1,13 +1,23 @@
 <configuration>
 
+  <conversionRule conversionWord="coloredLevel" converterClass="com.lightbend.lagom.internal.logback.ColoredLevel" />
+
   <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-    <!-- encoders are assigned the type
-         ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
     <encoder>
-      <pattern>%d{HH:mm:ss.SSS} XXX [%thread] %-5level %logger{36} - %msg%n</pattern>
+      <pattern>%date{"HH:mm:ss.SSS"} %coloredLevel %logger [%mdc] - %msg%n</pattern>
```

Comment:
The logging was messed up.

This is align the logging to what the Lagom docs suggest. And I took the opportunity to add logback.xml to the other services as well.

Reply:
nice